app/src/main/java/com/example/bookclub/SearchActivity.java

```diff
@@ -156,11 +157,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
             Toast.makeText(SearchActivity.this, "Logout Successfully!", Toast.LENGTH_SHORT).show();
             onLogout();
             return true;
+        }else if(id == R.id.action_library){
+            Toast.makeText(SearchActivity.this, "now in library", Toast.LENGTH_SHORT).show();
```

Comment:
This message is user visible so it can be added to strings.xml instead of having a literal.